### PR TITLE
Add subscriptions->streams to application-services

### PR DESCRIPTION
**Pull request by zanewoodfin**
I'm attempting to add a "Subscriptions" dropdown to Application Services.  It should Have a small Insights heading above it and one entry in the dropdown for "Streams for Apache Kafka".  The wireframe is [here](https://marvelapp.com/prototype/5dhc6c0/screen/82172335) for reference.  Let me know if this looks like it should generate what's in the wireframe.

### CODE REVIEW

File: chrome/application-services-navigation.json [hunk 1]
```diff
@@ -67,10 +67,28 @@
                 }
             ]
         },
+        {
+            "groupId": "insights",
```
Comment (Hyperkid123) [inline chrome/application-services-navigation.json:71]: You are missing the icon property. Based on the mocks, it should be
```js
"icon": "trend-up"
```

File: main.yml [hunk 2]
```diff
@@ -763,8 +763,12 @@ application-services:
       - id: service-registry
         title: "Service Registry"
         group: overview
-      - id: streams
-        group: overview
+      - id: subscriptions
```
Comment (Hyperkid123) [inline main.yml]: These changes are not required. Navigation is not using this config.

**Review by Hyperkid123 — COMMENTED** [review]
This is how the nav currently looks.

screenshot-prod foo redhat com_1337-2021 11 16-09_55_53

**Comment by cdcabrera** [comment]
Hey @Hyperkid123 is this file required to be updated with the app-services/subs/streams path?

https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/chrome/fed-modules.json#L291-L308

File: chrome/application-services-navigation.json (hunk 1 above)
Comment (zanewoodfin) [reply chrome/application-services-navigation.json:71]: Added.

File: main.yml (hunk 2 above)
Comment (zanewoodfin) [reply main.yml]: Reverted.

When would I need to update this location?

**Comment by Hyperkid123** [comment]
> Hey @Hyperkid123 is this file required to be updated with the app-services/subs/streams path?
> 
> https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/chrome/fed-modules.json#L291-L308

@cdcabrera sorry I was on PTO. Yes, you have to add a new entry to the modules in order to create a route in the applications services bundle. It can be the same shape as with the other two. But using the `/application-services/subscriptions` path.

**Review by Hyperkid123 — APPROVED** [review]

**Comment by Hyperkid123** [comment]
@zanewoodfin when is this supposed to be merged?

**Comment by zanewoodfin** [comment]
@Hyperkid123 I think this can be merged at any time because this is just going into ci-beta.

**Comment by mirekdlugosz** [comment]
Looks good :+1:

RHOSAK is already available on qa-beta and stage-beta, so from Subscriptions point of view, we would be fine with promoting this commit to other beta branches.

Screenshot from 2021-11-19 15-34-45

**Pull request merged**